bugs/lp-1802073.txt

```diff
@@ -3,13 +3,1692 @@ https://bugs.launchpad.net/ubuntu/+source/cloud-init/+bug/1802073
 
 === Begin SRU Template ===
 [Impact]
+Users on EC2 which run in 'Classic Networking' environments would
+lose network connectivity to their instance if they stopped and then
+started the instance due to EC2 changing the virtual nic MAC address.
+The change in configuration meant that the network config in the
+OS may not recognize the new interface and not configure networking.
+
 
 [Test Case]
+#!/bin/sh
```

Comment:
Some explanation of (and perhaps triple-ticking around) the two separate scripts would be good.